Civi/Funding/EventSubscriber/FundingCase/FundingCasePermissionsCacheClearSubscriber.php

```diff
@@ -79,12 +84,67 @@ public function preContact(PreEvent $event): void {
     }
   }
 
+  /**
+   * @throws \CRM_Core_Exception
+   */
+  public function onPreGroup(PreEvent $event): void {
```

Comment:
what does _onPre_ mean in this context? is being called before any update, create, delete action?